qiskit/execute_function.py

```diff
@@ -169,9 +169,9 @@ def execute(
             (provided the backend supports it). For OpenPulse jobs, only
             measurement level 2 supports this option. Default: False
 
-        max_credits (int): DEPRECATED This parameter is deprecated as of
-        Qiskit Terra 0.20.0, and will be removed in a future release. This parameter has
-        no effect on modern IBM Quantum systems, and no alternative is necessary.
+        max_credits (int): DEPRECATED This parameter is deprecated as of Qiskit Terra 0.20.0
+            and will be removed in a future release. This parameter has no effect on modern 
```

Comment:
You can also run `tox -eblack` to run black for code formatting. But I think it's probably just going to remove the trailing whitespace:

```suggestion
            and will be removed in a future release. This parameter has no effect on modern
```